infra/gcp/bash/ensure-prod-storage-gclb.sh

```diff
@@ -62,7 +62,7 @@ color 6 "Reconciling GCLB Backend Bucket"
 if ! gcloud --project "${PROD_PROJECT}" compute backend-buckets describe "${NAME}" >/dev/null 2>&1; then
   gcloud --project "${PROD_PROJECT}" compute backend-buckets create "${NAME}" --gcs-bucket-name="${BUCKET_NAME}"
 else
-  gcloud --project "${PROD_PROJECT}" compute backend-buckets update "${NAME}" --gcs-bucket-name="${BUCKET_NAME}"
+  gcloud --project "${PROD_PROJECT}" compute backend-buckets update "${NAME}" --gcs-bucket-name="${BUCKET_NAME}" --enable-cdn --cache-mode=FORCE_CACHE_ALL
```

Comment:
`FORCE_CACHE_ALL` defaults to a TTL of 1 hour (https://cloud.google.com/cdn/docs/caching), which is probably a fine starting point.

/lgtm

Reply:
@endocrimes targeting a TTL of 30 days for this. https://cloud.google.com/cdn/docs/using-ttl-overrides